sdk/storage/src/clients/storage_account_client.rs

```diff
@@ -0,0 +1,427 @@
+use azure_core::errors::AzureError;
+use azure_core::headers::*;
+use azure_core::HttpClient;
+use azure_core::EMPTY_BODY;
+use http::header::*;
+use http::method::Method;
+use http::request::{Builder, Request};
+use ring::hmac;
+use std::sync::Arc;
+use url::Url;
+
+pub(crate) const HEADER_VERSION: &str = "x-ms-version"; //=> [String] }
```

Comment:
What is this comment for?

Reply:
No reason. I don't even recall why it was introduced in the first place 🤷‍♂️ !

Reply:
Can we remove it?